code/game/objects/items/binoculars.dm

```diff
@@ -26,13 +26,12 @@
 	var/laser_cooldown = 0
 	var/cooldown_duration = 200 //20 seconds
 	var/obj/effect/overlay/temp/laser_target/laser
-	var/obj/effect/overlay/temp/laser_coordinate/coord
 	var/target_acquisition_delay = 100 //10 seconds
 	var/mode = 0 //Able to be switched between modes, 0 for cas laser, 1 for finding coordinates, 2 for directing railgun, 3 for orbital bombardment.
 	var/changable = TRUE //If set to FALSE, you can't toggle the mode between CAS and coordinate finding
 	var/ob_fired = FALSE // If the user has fired the OB
-	var/turf/current_turf // The target turf
-
+	var/turf/current_turf // The target turf, used for OBs
+	var/turf/target_turf // The target turf, used for mortars
```

Comment:
Can you be a dear and convert the var comments to autodoc please

Reply:
done